robyn/__init__.py

```diff
@@ -150,6 +151,26 @@ def start(self, url: str = "127.0.0.1", port: int = 8080):
         url = os.getenv("ROBYN_URL", url)
         port = int(os.getenv("ROBYN_PORT", port))
         open_browser = bool(os.getenv("ROBYN_BROWSER_OPEN", self.config.open_browser))
+        if is_port_in_use(port):
+            logger.error(
+                f"Failed to connect because port {port} is in use (or bad host)"
+            )
+            while True:
```

Comment:
One suggestion - 

We can change this to 

```python

while is_port_in_use(port):
       logger.error(....)
       
       question_port = [...]
       
       ....
```

Reply:
fixed 🍝